conformance/conf_tests/helpers.py

```diff
@@ -21,56 +21,114 @@
 from pygame.locals import SRCALPHA
 
 
-def create_surface():
+DEPTHS = (8, 16, 24, 32)
+
+
+class conformance_test_case(object):
+    """Tacks the appropriate attributes onto the test function"""
+
+    registry = {}
+
+    def __init__(self, test_file_name, depths=DEPTHS):
+        self.test_file_name = test_file_name
```

Comment:
Could we get the filename from the function being wrapped by stripping `test_` from the start of it?